packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -1,18 +1,199 @@
+import { ECIES, PhoneNumberUtils, SignatureUtils } from '@celo/utils'
+import { zip3 } from '@celo/utils/lib/collections'
 import BigNumber from 'bignumber.js'
+import * as Web3Utils from 'web3-utils'
+import { CeloToken } from '../base'
 import { Attestations } from '../generated/types/Attestations'
-import { BaseWrapper, proxySend } from './BaseWrapper'
+import { BaseWrapper, proxyCall, proxySend, toNumber, tupleParser, wrapSend } from './BaseWrapper'
+const parseSignature = SignatureUtils.parseSignature
 
 export interface AttestationStat {
   completed: number
   total: number
 }
 
+export enum AttestationState {
+  None,
+  Incomplete,
+  Complete,
+}
+
+export interface ActionableAttestation {
+  issuer: string
+  attestationState: AttestationState
+  time: number
+  publicKey: string
+}
+
+const parseAttestationInfo = (rawState: { 0: string; 1: string }) => ({
+  attestationState: parseInt(rawState[0], 10),
+  time: parseInt(rawState[1], 10),
+})
+
+function attestationMessageToSign(phoneHash: string, account: string) {
+  const messageHash: string = Web3Utils.soliditySha3(
+    { type: 'bytes32', value: phoneHash },
+    { type: 'address', value: account }
+  )
+  return messageHash
+}
+
 export class AttestationsWrapper extends BaseWrapper<Attestations> {
+  /**
+   *  Returns the time an attestation can be completable before it is considered expired
+   */
+  attestationExpirySeconds = proxyCall(
+    this.contract.methods.attestationExpirySeconds,
+    undefined,
+    toNumber
+  )
+
+  /**
+   * Returns the attestation stats of a phone number/account pair
+   * @param {string} phoneNumber Phone Number
+   * @param {string} account Account
+   */
+  getAttestationStat = proxyCall(
```

Comment:
try hover on this method.. I'm sure it's going to say `arg_0, arg_1` since they are extracted from `tupleParser`. In this case, what i did is `getBuyTokenAmount: (sellAmount: NumberLike, sellGold: boolean) => Promise<BigNumber> `

to make them explicit